en/SUMMARY.md

```diff
@@ -70,6 +70,7 @@
   * [Advanced Controller Orientation](advanced_config/advanced_flight_controller_orientation_leveling.md)
   * [Full Parameter Reference](advanced_config/parameter_reference.md)
 * [Peripheral Hardware](peripherals/README.md)
+  * [Companion Computer Peripherals](peripherals/companion_computer_peripherals.md)
```

Comment:
Something very weird is going on - the left side "master" is nothing like the PX4 master branch TOC, so this addition will fail. Can you resolve the conflict (you can do this from the front page of this PR via the Github GUI) basically just use what is in master, and add this new page to the end.